packages/DataTable/src/components/RowHeight/RowHeight.js

```diff
@@ -61,6 +61,7 @@ export default function RowHeight(props) {
 
 RowHeight.propTypes = propTypes;
 RowHeight.defaultProps = defaultProps;
+RowHeight.displayName = "DataTable.Navigation.RowHeight";
```

Comment:
Why you changed the name? This is rendered on the Navigation but the logic doesn't belong to the Navigation, also if you change this here you should change it on the `DataTable.js` where use it to extract the children with extract children.

Reply:
Maybe `DataTableControls.RowHeight` ? or do you prefer `DataTable.RowHeight` ? I understand `RowHeight` means the row height of the table, but this component is for changing the row height, the table itself doesn't care the value of row height. Actually if i remove this part https://github.com/acl-services/paprika/pull/300/files#diff-ccd22f487246440a452bd36678a0f647R23 I don't really need to know the display name of the component